plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.ts

```diff
@@ -62,6 +69,9 @@ export class GoogleAnalytics implements AnalyticsApi {
   static fromConfig(config: Config) {
     // Get all necessary configuration.
     const trackingId = config.getString('app.analytics.ga.trackingId');
+    const gaAddress =
+      config.getOptionalString('app.analytics.ga.gaAddress') ??
+      'https://www.google-analytics.com/analytics.js';
```

Comment:
Perhaps safer to just keep passing on `undefined` here right? In case they change the location

Reply:
makes sense!